Resources/Prototypes/Entities/Objects/Specific/syndicate.yml

```diff
@@ -77,6 +77,11 @@
     preset: StorePresetUplink
     balance:
       Telecrystal: 0
+  - type: Prayable
+    sentMessage: prayer-popup-notify-synducate-sent
```

Comment:
```suggestion
    sentMessage: prayer-popup-notify-syndicate-sent
```
you should add a locale string, with message what would popup

Reply:
Should I add the ability for only the aplink owner to use the call?